### PR TITLE
Fix OpenShift 3 and 4 tests

### PR DESCRIPTION
**Pull request by phracek**
This pull request updates container-common-scripts.

And fixes OpenShift 3 and OpenShift 4 tests

Signed-off-by: Petr "Stone" Hracek <phracek@redhat.com>

### CODE REVIEW

**Comment by phracek** [comment]
[test-all]

**Comment by phracek** [comment]
The tests are failing because of image stream for centos7 does not contain the latest version.
It is fixed by the second commit.

[test-all]

**Comment by phracek** [comment]
@yselkowitz Can you please look at imagestreams in this pull request, If they are fine? Thank you

[test-all]

**Comment by phracek** [comment]
[test-all]

**Comment by phracek** [comment]
[test-all]

File: imagestreams/mariadb-centos.json [hunk 1]
```diff
@@ -14,13 +14,13 @@
...
           "openshift.io/provider-display-name": "Red Hat, Inc.",
-          "description": "Provides a MariaDB database on CentOS. For more information about using this database image, including OpenShift considerations, see https://github.com/sclorg/mariadb-container/tree/master/10.3/README.md.\n\nWARNING: By selecting this tag, your application will automatically update to use the latest version of MariaDB available on OpenShift, including major version updates.",
+          "description": "Provides a MariaDB database on CentOS. For more information about using this database image, including OpenShift considerations, see https://github.com/sclorg/mariadb-container/tree/master/10.5/README.md.\n\nWARNING: By selecting this tag, your application will automatically update to use the latest version of MariaDB available on OpenShift, including major version updates.",
           "iconClass": "icon-mariadb",
           "tags": "database,mariadb"
         },
         "from": {
           "kind": "ImageStreamTag",
-          "name": "10.3-el8"
+          "name": "10.5-el8"
```
Comment (yselkowitz) [inline imagestreams/mariadb-centos.json:23]: There is no 10.5 for centos8. Do you mean the recently added `10.5-el7`?

**Review by yselkowitz — CHANGES_REQUESTED** [review]

File: imagestreams/mariadb-centos.json (hunk 1 above)
Comment (phracek) [reply imagestreams/mariadb-centos.json:23]: Good point. We will have soon quay.io/centos7/mariadb-105-centos7.

**Comment by phracek** [comment]
This can be merged as soon as https://github.com/sclorg/mariadb-container/pull/159 is merged.

File: imagestreams/mariadb-centos.json (hunk 1 above)
Comment (phracek) [reply imagestreams/mariadb-centos.json]: Fixed

**Comment by phracek** [comment]
[test-all]

**Comment by phracek** [comment]
rebased based base on the #159

**Comment by phracek** [comment]
[test-all]

**Comment by phracek** [comment]
@pkubatrh @zmiklank The whole test suite is passing. Please take a look at it as you have time. Thank you

**Review by zmiklank — APPROVED** [review]
LGTM

**Comment by zmiklank** [comment]
@phracek Maybe just one question - the name of the PR says, that it is about openshift3, but the file changed is ` test/run-openshift-remote-cluster`, which serves as openshift4 runner, if I understand it correctly. Why is that?

**Pull request merged**